renovate.json

```diff
@@ -0,0 +1,6 @@
+{
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "extends": [
+        "config:base"
+    ]
+}
```

Comment:
Delete. I hate these dependency update bots because they spam you constantly. Why they don't default to a weekly/monthly combined update beats me.

Reply:
We can configure it to update weekly.

Reply:
Hmmm, for now let's not do this. If I was actively developing the project, I'd say yes but it's been in maintenance mode for a few years now and I'd rather just leave things alone as long as they work.

Revert and then I'll merge! Thanks for doing this!

Reply:
070470784aad5f1cd6231e7cfddb340fdc88836d